api/v1alpha1/helmchartproxy_webhook.go

```diff
@@ -37,19 +37,19 @@ var helmchartproxylog = logf.Log.WithName("helmchartproxy-resource")
 func (r *HelmChartProxy) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewWebhookManagedBy(mgr).
 		For(r).
+		WithDefaulter(r).
```

Comment:
@mboersma What do you think about moving web-hooks to the separate package and exclude it from the apidiff calculation to avoid confusion?